### PR TITLE
Split code out into calendar addon

### PR DESCRIPTION
**Pull request by jrjohnson**
All of this code is now in the ilios/calendar add-on which can be used both by the fronted and by the embedded app.

### CODE REVIEW

File: app/locales/en/translations.js [hunk 1]
```diff
@@ -294,6 +294,7 @@ export default {
     'month': 'Month',
     'today': 'Today',
     'backToCalendar': 'Back to Calendar',
+    'loadingEvents': 'Loading Events...',
```
Comment (jrjohnson) [inline app/locales/en/translations.js]: @saschaben / @dartajax please provide a translation for `Loading Events...`;

Comment (saschaben) [inline app/locales/en/translations.js]: I think for the french it will be best to use `Rassembler les événements...` ("getting events...")

Comment (dartajax) [inline app/locales/en/translations.js]: Cargando eventos ...

**Comment by jrjohnson** [comment]
This is ready for review

**Comment by jrjohnson** [comment]
@thecoolestguy I rebased, but tests should run again just in case.

**Pull request merged**